### DISCUSSION TITLE
I made a 7 hours music video with Toonsquid

### DISCUSSION THREAD

**Discussion by overcesium** [keiwando/toonsquid#688] [Show and tell]
This summer I made in collaboration with a famous italian comic artist a 7 minute animation looped for 7 hours. 
We only used Toonsquid! 
At Agouti Studio we've been using this amazing software almost daily for the past year and I can safely say that we don't miss using the computer for animating!

[Here's the video](https://youtu.be/qOXezaI7YdQ?si=K_fhrR2wk54CQjXB)

**Comment by budaloops** [comment]
lllove your design style ☺️🌟

**Comment by watergirl2** [comment]
Love it. Good for me to learn Italian.

**Comment by otaworks** [comment]
Woah! You weren't kidding when you said 7 hours. I thought you meant you used 7 hours to work on the music video
